src/Controller/ModulesController.php

```diff
@@ -366,6 +366,12 @@ public function relationshipsJson($id, string $relation) : void
             }
 
             $response = $this->apiClient->get($available);
+
+            foreach ($response['data'] as $index => $obj) {
+                if ($obj['id'] == $id) {
+                    array_splice($response['data'], $index, 1);
```

Comment:
Why removing data from `$response['data']`? a comment would help in here ;)

Reply:
right, i'll comment it (anyway is cause in folder tree i want to remove the element at index $index, which is the current object)